standard_releases.md

```diff
@@ -1,59 +1,215 @@
+  translation. The process for making these changes is:
+  * First create an issue in Github to track the problem.
+  * Then create a new branch that uses the issue number, eg.
+    `297-concept-translation`.
+  * When the branch is ready to be merged into master, pull the strings from
+    Transifex and push them to the translation branch eg `v0-2-0-translation`.
+  * Now merge the branch (297-concept-translation) into master before pushing
+    the updated docs to Transifex.
+  * Check for any changes in Transifex and use the translation branch
+    (`v0-2-0-translation`) to replace any translations that have been
+    overwritten.
+  * Once the translation is finalised, merge the translation branch into
+    `master`.
+* Once the release is finalised and everything is ready, create a branch for the
+  new version (eg, v0-2-0) from `master`.
+* Make the new version live on ReadTheDocs.
+
+### A new language to the current release
+
+Until such time that we support several stable releases, our assumption is that
+only the current release (and any future ones) will be translated into new
+languages.
+
+When there are no significant differences between `master` and the current
+release, the process should be to use the existing project in Transifex. For
+example, BODS v0-2-0 has a project called "BODS v0.2" in Transifex. New
+languages for translation can be added at any time through the Transifex UI.
+
+* Make a new branch from `master` for the translation work
+* Review, test, snag and finally merge this branch into `master`
+* Make a new branch from the current release and bring across the new
+  translations from Transifex.
+* As before, review this branch and fix any snagging issues with the translation
+  in this context. It is assumed that normally these fixes will not need to be
+  merged back into master, but if they uncover bugs not previously found, those
+  should be `git cherry-pick`-ed back into `master`.
+
+When `master` and the current release have diverged significantly (a situation
+we've yet to encounter), it's assumed that the correct process will be to treat
+the translation of each as a separate project. In this case, it's probably most
+helpful to translate the current release first, bring across any fixes and only
+initiate translation of `master` when it's due for release.
+
+### Fixes to an existing release
+
+When a bug is discovered or a correction needs to be made to an existing
+release and it is deemed severe enough to require an immediate fix (i.e.
+it cannot wait for the next naturally-occurring release), then we need
+to make an unplanned release. The process for this is as follows:
+
+* Make a new branch named something like `v0-2-0-description-of-issue`
+  from the current release branch and commit the fix(es) to it
+* Review and test this branch in a pull-request
+* If necessary, perform necessary translation work on this branch once this fix
```

Comment:
Do we want to maintain copies, in Transifex, of both versions of the standard? ODS have a method of uploading a new project so that previously translated sections are maintained. This is currently undocumented.

Reply:
I think we will have to maintain copies once we have more than one version with translations (it's only v0.2 at the moment, right?), but probably not worth documenting until we've had to do it for real?

Reply:
Sorry, I wasn't very clear. I meant to refer to the case where we create v0-2-1 (for example). In Transifex, should we create a new project for this or do we edit v0-2?
I don't think we have any limit on the number of projects that we can have in Transifex.

Reply:
I think where we have a patch change - would should be 'compatible' whatever that might mean - we don't need to. My reasoning is that once 0.2.1 exists, we're never going to be making any changes to translations in 0.2, we'd change them in 0.2.1 to make 0.2.2. Therefore the v0-2 Transifex project can be thought of as being the translation of the latest patch release. Renaming the project to 0.2.x might help make that clearer?

Where's incompatibility - which I think we still need some examples to define, but we can assume means either a minor or major version number change, 0.3 or 1.0 - we probably do need a new project. 

If you're happy with that reasoning, I can add that in too?